recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --root=/
```

Comment:
These reviews are death by 1000 cuts.  It would be much nicer to put up PRs, or add something to the linter so that it provides a link to a "fixed" file.  Making people go back and forth, and keep track of each little fix (which are often pedantic and of questionable functional value) is asking a lot people who have a lot to do.